api/tests/scripts/external_users/unstack_batch_cloud_task_queue_test.py

```diff
@@ -33,7 +33,7 @@ def test_run_task_batch(mock_get_client, mock_update_users_attributes, users_cou
     users = BeneficiaryGrant18Factory.create_batch(users_count)
     mock_get_client.return_value.list_tasks.return_value = [MockTask(f"task_{user.id}", user.id) for user in users]
 
-    tasks, deleted_tasks = unstack_batch_queue("some_queue_name", chunk_size)
+    tasks, deleted_tasks = unstack_batch_queue("some_queue_name", chunk_size, sleep_time=0)
```

Comment:
Pourrait-on modifier le délai au niveau de la fonction, afin de la passer à 0 quand `settings. IS_RUNNING_TESTS is True` ?
Cela évitera que d'autres tests se fassent _endormir_.

Reply:
J'ai hésité mais je préfère jouer la sureté et ne pas l'ajouter.

Reply:
Au  passage, je n'ai pas trouvé d'autre test qui soit ralenti ici